subprojects/packagefiles/glfw/meson_options.txt

```diff
@@ -0,0 +1,26 @@
+#####  GLFW build options
+
+#@ Build options
+option('build-docs',
+        type : 'boolean',
+        value : false,
+        description : 'Build GLFW documentation?')
+option('build-tests',
+        type : 'boolean',
+        value : false,
+        description : 'Build GLFW test programs?')
+option('build-examples',
+        type : 'boolean',
+        value : false,
+        description : 'Build GLFW example programs?')
+option('install',
+        type : 'boolean',
+        value : false,
+        description : 'Install GLFW?')
```

Comment:
I am not sure I understand the need for this option, can you please elaborate?

Reply:
If a user wants to install the library, its headers, and the various examples, they should be able to choose to do so, **even if** it is a subproject.